cmd/multi-factor-auth/swagger/data.go

```diff
@@ -223,3 +223,60 @@ type _ struct {
 	// required:true
 	Payload requests.SignMultipleTransactions
 }
+
+// swagger:route POST /set-security-mode-no-expire Guardian setSecurityModeNoExpireRequest
+// Sets the security mode permanently.
+// This request does not need the Authorization header
+//
+// responses:
+// 200: setSecurityModeNoExpireMessageResponse
+
+// The status of the operation
+// swagger:response setSecurityModeNoExpireMessageResponse
+type _ struct {
+	// in:body
+	Body struct {
+		// SetSecurityModeNoExpireMessageResponse
+		// HTTP status code
+		Code string `json:"code"`
+		// Internal error
+		Error string `json:"error"`
+	}
+}
+
+// swagger:parameters setSecurityModeNoExpireRequest
```

Comment:
can be renamed to securityModeNoExpireRequest, reused for the next one and last structure should be deleted

Reply:
updated